src/app/button/terra-button.component.ts

```diff
@@ -67,11 +71,32 @@ export class TerraButtonComponent
         this.inputIsHighlighted = false;
     }
 
+    ngOnInit():void
+    {
+        this._currentTooltipPlacement = this.inputTooltipPlacement;
+    }
+
     private click(event:Event):void
     {
         if(isNullOrUndefined(this.inputIsDisabled) || this.inputIsDisabled === false)
         {
             this.outputClicked.emit(event);
         }
     }
+
+    private setTooltipPlacement(event:MouseEvent):void
```

Comment:
check mal ob der immer da anhält, ich hab nämlich die erfahrung gemacht, dass er bei mousemove immer in die methoden reingesprungen ist und das nicht nur einmal... einfach einen haltepunkt machen

Reply:
Danke für den Hinweis!
Der Name (mousemove) sagt bereits, dass das Event bei jeder Mausbewegug ausgelöst wird.
mouseenter wird jedoch nur bei jedem hovern über den Button einmal ausgelöst.
Beim Test ist mir jedoch aufgefallen, dass die Funktion trotzdem etwas zu häufig ausgeführt wird. Ich untersuche das